_posts/2017-10-23-pair-programming-with-perry.md

```diff
@@ -0,0 +1,91 @@
+    <br><br>
+    As far as code comprehension goes, pairing results in two people having a clear understanding of the code that was just written as opposed to the usual one. This is more of a long-term gain because there is always the possibility that the person that worked on the code will not be available to help explain things, so having an additional person with that same knowledge could pay off big down the road.
+</div>
+<br>
+<span class='os-green'>*Alright, Perry, you seem to really advocate pair programming. Are there any cons to pairing?*</span>
+<div style='margin-left: 2em;' >
+    There definitely are, though I think the good outweigh the bad by a long shot.
+    <br><br>
+    Pairing forces you to sit next to someone and interact, and some days you don’t want to; you’d much rather fly solo at your desk with your headphones on, which is totally fine, but the forced interaction can be seen as a drawback.
+    <br><br>
+    The possibility is always there that you’ll butt heads with your pair, which isn’t fun for anybody. However, I think this offers a chance to improve interpersonal skills as you’ll learn to compromise and communicate more effectively.
+    One last negative I’ll note is that some believe pairing reduces your team’s velocity. This isn’t something I agree with. I believe pairing pays short term and long term dividends with all of the pros I talked about before.
+</div>
+<br>
+<span class='os-green'>*That’s something that came to my mind right away is that ‘I have to sit and talk to somebody all day?!’ Doesn’t sound so fun to me.*</span>
+<div style='margin-left: 2em;' >
+    Right, and there is nothing wrong with asking your pair if it’s cool to fly solo for the day. We’re all developers and want to just code by ourselves sometimes.
+</div>
+<br>
+<span class='os-green'>*Do you have any final tips for someone that hasn’t paired before but would like to start?*</span>
+<div style='margin-left: 2em;' >
+    Create a comfortable environment for each of you. I recommend you each have your own keyboard for starters, as well as an external monitor or two, just avoid using only your laptop screen.
+    <br><br>
+    It’s important that both of you are engaged in the problem or solution. If you feel like you’ve been spinning your wheels for a while then recommend to your pair that you get up and take a break and come back fresh. Ask your pair questions about what they’re writing or thinking about the problem. Try to avoid long silent stretches where one person is typing while the other is just watching as that’s defeating the whole point of pairing.
+    <br><br>
+    Pairing takes practice, just like any other skill, so don’t be discouraged if it doesn’t go super smooth the first time or even the first few times you try it. You’ll probably find that you pair better with certain devs, and there’s nothing wrong with that.
```

Comment:
*'... the first time, or even the first few times you try it.'

Reply:
Most of these are just suggestions rather than corrections.  I think the conversational nature of this blog post gives you a lot of flexibility.. so for example, not every sentence needs to be grammatically correct.  In the end it's your call on most if not all of this.

Good job!

Reply:
Thank you for all of this quality feedback. I greatly appreciate it and used all of your suggestions.